modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-cluster-enabling-features.adoc
+
+[id="nodes-cluster-enabling-features-about_{context}"]
+= Understanding feature gates
+
+You can use the Feature Gate custom resource (CR) to enable specific feature sets in your cluster. A feature set is a collection of {product-title} features that are not enabled by default. 
+
+For example, the `TechPreviewNoUpgrade` feature set allows you to enable a subset of the current Technology Preview features on test clusters, where you can fully test them, while leaving the features disabled on production clusters.
+
+You can activate any of the following feature sets by using the Feature Gate CR:
+
+[options="header"]
+|===
+| Feature Set| Description
+
+|`IPv6DualStackNoUpgrade`
+|Enables the dual-stack networking mode in your cluster. Dual-stack networking supports the use of IPv4 and IPv6 simultaneously. Enabling this feature set is *not supported*, cannot be undone, and prevents upgrades. This feature set is not recommended on production clusters.
+
+|`TechPreviewNoUpgrade`
+a|Enables Technology Preview features that are not part of the default features. Enabling this feature set cannot be undone and prevents upgrades. This feature set is not recommended on production clusters. 
+
+// |`LatencySensitive`
+// |Enables the Topology Manager in your cluster. Upgrades are enabled with this feature.
+
+The following Technology Preview features are enabled by this feature set:
+
+* link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation[RotateKubeletServerCertificate]. Enables the rotation of the server TLS certificate on the kubelet.
+* SupportPodPidsLimit. Enables limiting process IDs (PIDs) in pods.
```

Comment:
@lbarbeevargas That is the name of a feature in the feature set. I don't think we mark-up feature names(??). We don't in Tech Preview notes, apparently: https://github.com/openshift/openshift-docs/pull/13878/files#diff-99eeb6425c261c0f96653039df58187866a5429c67f34e40c6364f0bd3d5e6a0L10

Reply:
@mburke5678 The feature names caught my attention here because they are in PascalCase, whereas the feature name (`:FeatureName:` variable) example in the guidelines is spelled out.

Reply:
@lbarbeevargas The Pascal case spelling is the [common usage on the internet](https://www.google.com/search?client=firefox-b-1-e&q=SupportPodPidsLimit). But, the Kubernetes docs use [Pascal case in the feature gates docs](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#overview) and Pod PID limits in the [release notes](https://kubernetes.io/docs/concepts/policy/pid-limiting/).  
What if we go with both, too? `Pod PID limits (SupportPodPidsLimit)`

Reply:
I haven't come across this situation before, but LGTM if it makes sense to the other reviewers. 👍 If going with the dual format, the first bullet can be updated as well:

> link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation[Certificate rotation (RotateKubeletServerCertificate)]. Enables the rotation of the server TLS certificate on the kubelet.